### PR TITLE
Run PHPCBF to fix PSR-2 violations.

### PR DESCRIPTION
**Pull request by ganoze**
<h1> Pull Request Template </h1>

<h2> Description </h2>

Automatically fix PSR-2 style **errors** (not warnings) with PHPCBF.

`phpcbf -n -p --extensions=php --parallel=2 --standard=PSR2 .`

<h2> Type of change </h2>

Please delete options that are not relevant.

- [X] Patch fixing an issue (non-breaking change)

<h2> Target serie </h2>

Please delete series that are not relevant.

- [X] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Not relevant.

<h2> Checklist </h2>

<h5> Community contributors & Centreon team </h5>

- [X] I followed the **coding style guidelines** provided by Centreon
- [ ] ~I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)~
- [ ] ~I have commented my code, especially **hard-to-understand areas** of the PR.~
- [ ] ~I have made corresponding changes to the **documentation**.~
- [X] I have **rebased** my development branch on the base branch (master, maintenance).

<h5> Centreon team only </h5>

- [ ] ~I have made sure that the **unit tests** related to the story are successful.~
- [ ] ~I have made sure that **unit tests covers 80%** of the code written for the story.~
- [X] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: lib/Slug.class.php [hunk 1]
```diff
@@ -30,16 +30,16 @@
  */
 class Slug implements ArrayAccess
 {
-  protected $original = null;
-  protected $slug = null;
-  protected $options = array(
+    protected $original = null;
+    protected $slug = null;
+    protected $options = array(
     'to_lower'        => true,
```
Comment (sc979) [inline lib/Slug.class.php]: odd indentation

File: lib/Slug.class.php [hunk 2]
```diff
@@ -30,16 +30,16 @@
...
     'max_length'      => null,
     'prefix'          => null,
     'postfix'         => null,
     'seperator_char'  => '-'
-  );
-  protected $char_map = array(
+    );
+    protected $char_map = array(
     'Š'=>'S',
```
Comment (sc979) [inline lib/Slug.class.php]: odd indentation

File: lib/Slug.class.php [hunk 3]
```diff
@@ -110,7 +110,7 @@ class Slug implements ArrayAccess
     'ƒ'=>'f',
     'Ŕ'=>'R',
     'ŕ'=>'r'
-  );
+    );
   /**
```
Comment (sc979) [inline lib/Slug.class.php]: odd indentation

File: lib/Slug.class.php [hunk 4]
```diff
@@ -126,146 +126,134 @@ class Slug implements ArrayAccess
...
     {
-      $this->char_map = $char_map;
+        $this->original = $original;
+        $this->options = array_merge($this->options, $options);
+        if (null !== $char_map) {
+            $this->char_map = $char_map;
+        }
     }
-  }
   /**
```
Comment (sc979) [inline lib/Slug.class.php]: odd indentation

File: src/CentreonRemote/Tests/Infrastructure/Export/ExportManifestTest.php [hunk 5]
```diff
@@ -82,23 +82,23 @@ protected function setUp()
         //->expects($this->any())
         $this->manifest
             ->method('removePath')
-            ->will($this->returnCallback(function() {
+            ->will($this->returnCallback(function () {
                     $args = func_get_args();
```
Comment (sc979) [inline src/CentreonRemote/Tests/Infrastructure/Export/ExportManifestTest.php]: odd indentation

File: www/api/class/centreon_configuration_service.class.php [hunk 6]
```diff
@@ -122,7 +122,8 @@ public function getList()
         if (isset($this->arguments['s'])) {
             $sTypeList = array('s', 'm', 'all');
             if (in_array(strtolower($this->arguments['s']), $sTypeList)) {
-                $s = $this->arguments['s'];;
+                $s = $this->arguments['s'];
+                ;
```
Comment (sc979) [inline www/api/class/centreon_configuration_service.class.php]:
```suggestion
```

File: www/include/common/mobile_menu.php [hunk 7]
```diff
@@ -4,14 +4,18 @@
 </header>
 <nav class="nav-drill">
     <ul class="nav-items nav-level-1">
-        <?php foreach ($treeMenu as $index => $subMenu): ?>
-        <li class="nav-item <?php if(!empty($subMenu['children'])): ?>nav-expand<?php endif; ?>">
-            <a class="nav-link <?php if(!empty($subMenu['children'])): ?>nav-expand-link<?php endif; ?>" href="#">
+        <?php foreach ($treeMenu as $index => $subMenu) : ?>
+        <li class="nav-item <?php if (!empty($subMenu['children'])) :
+?>nav-expand<?php
```
Comment (sc979) [inline www/include/common/mobile_menu.php]: please -> don't apply PSR2 on html code

File: www/include/core/footer/footer.php [hunk 8]
```diff
@@ -62,7 +62,8 @@
                     <a href="https://www.centreon.com" title='Centreon Services Overview' target='_blank'>Centreon</a> |
                     <a href="https://github.com/centreon/centreon.git" title='Follow and Fork us on Github' target='_blank'>Github Project</a> |
                     <a href="https://centreon.github.io" title='Give us your feedback' target='_blank'>Slack</a>
-                    <?php if (!empty($centreon->optGen["centreon_support_email"])) { ?> |
+                    <?php if (!empty($centreon->optGen["centreon_support_email"])) {
+?> |
```
Comment (sc979) [inline www/include/core/footer/footer.php]: pls->don't apply PSR2 on html

File: www/include/monitoring/common-Func.php [hunk 9]
```diff
@@ -196,15 +196,13 @@ function get_notified_infos_for_service($serviceId, $hostId)
             "service_template_model_stm_id FROM service WHERE service_id = " . $serviceId;
         $DBRESULT = $pearDB->query($query);
         $contactAdd = $DBRESULT->fetchRow();
-        if (
-            !isset($contactAdd['service_template_model_stm_id']) ||
+        if (!isset($contactAdd['service_template_model_stm_id']) ||
             is_null($contactAdd['service_template_model_stm_id'])
             || $contactAdd['service_template_model_stm_id'] == ''
```
Comment (sc979) [inline www/include/monitoring/common-Func.php]:
```suggestion
            $contactAdd['service_template_model_stm_id'] == ''
```

File: www/include/monitoring/common-Func.php [hunk 10]
```diff
@@ -196,15 +196,13 @@ function get_notified_infos_for_service($serviceId, $hostId)
             "service_template_model_stm_id FROM service WHERE service_id = " . $serviceId;
         $DBRESULT = $pearDB->query($query);
         $contactAdd = $DBRESULT->fetchRow();
-        if (
-            !isset($contactAdd['service_template_model_stm_id']) ||
+        if (!isset($contactAdd['service_template_model_stm_id']) ||
             is_null($contactAdd['service_template_model_stm_id'])
```
Comment (sc979) [inline www/include/monitoring/common-Func.php]:
```suggestion
             is_null($contactAdd['service_template_model_stm_id']) ||
```

File: www/include/monitoring/status/Common/commonJS.php [hunk 11]
```diff
@@ -350,34 +350,34 @@ function construct_HostGroupSelectList(id) {
...
-    if ($centreon->user->access->admin || ($centreon->user->access->admin == 0 && isset($hgBrk[$hostgroups["name"]]))) {
-        if (!isset($tabHG)) {
-            $tabHG = array();
-        }
-        if (!isset($tabHG[$hostgroups["name"]])) {
-            $tabHG[$hostgroups["name"]] = "";
-        } else {
-            $tabHG[$hostgroups["name"]] .= ",";
+        while ($hostgroups = $DBRESULT->fetchRow()) {
+            if ($centreon->user->access->admin || ($centreon->user->access->admin == 0 && isset($hgBrk[$hostgroups["name"]]))) {
```
Comment (sc979) [inline www/include/monitoring/status/Common/commonJS.php]: isn't this line too long ?

File: www/include/monitoring/status/Common/commonJS.php [hunk 12]
```diff
@@ -350,34 +350,34 @@ function construct_HostGroupSelectList(id) {
...
-while ($hostgroups = $DBRESULT->fetchRow()) {
-    if ($centreon->user->access->admin || ($centreon->user->access->admin == 0 && isset($hgBrk[$hostgroups["name"]]))) {
-        if (!isset($tabHG)) {
-            $tabHG = array();
-        }
-        if (!isset($tabHG[$hostgroups["name"]])) {
-            $tabHG[$hostgroups["name"]] = "";
-        } else {
-            $tabHG[$hostgroups["name"]] .= ",";
+        while ($hostgroups = $DBRESULT->fetchRow()) {
```
Comment (sc979) [inline www/include/monitoring/status/Common/commonJS.php]: wrong indentation -> the problem comes from line 348. And seems to impact all the lines until line 380

File: www/include/options/oreon/modules/licenseUpload.php [hunk 13]
```diff
@@ -51,7 +51,8 @@
 } else {
     // Directory for put license files
     $licensePath = '/etc/centreon/license.d/';
-    $destination = $licensePath . $licenseModuleName . $extensionFileLicense;;
+    $destination = $licensePath . $licenseModuleName . $extensionFileLicense;
+    ;
```
Comment (sc979) [inline www/include/options/oreon/modules/licenseUpload.php]:
```suggestion
```

File: www/install/upgrade.php [hunk 14]
```diff
@@ -38,11 +38,10 @@
  * configuration
  */
 require_once realpath(dirname(__FILE__) . "/../../config/centreon.config.php");
```
Comment (sc979) [inline www/install/upgrade.php]:
```suggestion
 require_once realpath(__DIR__ . "/../../config/centreon.config.php");
```

Comment (sc979) [reply www/install/upgrade.php]: as it comes out. Maybe time to change this too ;)

File: www/lib/HTML/QuickForm/advmultiselect.php [hunk 15]
```diff
@@ -390,117 +394,139 @@ function __construct($elementName = null, $elementLabel = null,
...
-            return PEAR::throwError('Argument 1 of HTML_QuickForm_advmultiselect::' .
+                } else {
+                    $this->_updateAttrArray(
+                        $this->_bottomButtonAttributes,
+                        $this->_parseAttributes($attributes)
+                    );
+                }
+                break;
+            default;
+            return PEAR::throwError(
```
Comment (sc979) [inline www/lib/HTML/QuickForm/advmultiselect.php]: we don't use PEAR anymore (but it's in the quickform lib).

Comment (ganoze) [reply www/lib/HTML/QuickForm/advmultiselect.php]: This PR is for style fix only. I do not know how to replace this and IMHO this should be part of another PR.

**Pull request merged**